RealmSwift-swift2.0/List.swift

```diff
@@ -313,7 +313,22 @@ extension List: ExtensibleCollectionType {
         return RLMGenerator(collection: _rlmArray)
     }
 
-    // MARK: ExtensibleCollection Support
+    // MARK: RangeReplaceableCollection Support
+
+    /**
+    Replace the given `subRange` of elements with `newElements`.
+
+    :param: subRange    The range of elements to be replaced.
+    :param: newElements The new elements to be inserted into the list.
+    */
+    public func replaceRange<C : CollectionType where C.Generator.Element == T>(subRange: Range<Int>, with newElements: C) {
```

Comment:
style-wise, we've been keeping the colon right next to the type (`C: CollectionType where ...`)